.coveragerc

```diff
@@ -1,6 +1,10 @@
 [run]
 branch = True
-source = piptools
+source = .
+include = piptools/*
```

Comment:
Just `piptools` might work better (also for report, but uses `run` as default IIRC).
It uses the module then.

Reply:
I've tried `include = piptools`, but got "No data to report."

Reply:
Weird.  It says for me:

> Coverage.py warning: --include is ignored because --source is set (include-ignored)

And it seems `include` with `run` can be removed.  Seems to be fine with `report`.